darts/models/forecasting/arima.py

```diff
@@ -32,10 +32,17 @@ class ARIMA(TransferableFutureCovariatesLocalForecastingModel):
     def __init__(
         self,
         p: int = 12,
-        d: int = 1,
-        q: int = 0,
-        seasonal_order: Tuple[int, int, int, int] = (0, 0, 0, 0),
-        trend: Optional[str] = None,
+        d: Union[int, Iterable[int]] = 1,
+        q: Union[int, Iterable[int]] = 0,
+        seasonal_order: Tuple[
```

Comment:
You can also define this type at the top of the script and use it here. Because now it is not so readable.

Reply:
I think this is the most readable way. Having a type alias at the top will force a super descriptive name that could be less clear to the end user.

Reply:
However, I do agree that seasonal_order is quite hard to grasps. I will check what I can do.